### PR TITLE
Added changes to include deeplink to meeting side panel

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/tab-deeplink/csharp/DeepLinkBot/Services/DeeplinkHelper.cs [hunk 1]
```diff
@@ -25,6 +25,22 @@ public string GetDeepLinkToTabTask(string teamsUrl, string appID,string entityId
             return channelDeepLink;
         }
 
+        public string GetDeepLinkToMeetingSidePanel(string teamsUrl, string appID, string baseUrl, string entityId, string chatId, string contextType)
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/csharp/DeepLinkBot/Services/DeeplinkHelper.cs:28]: Kindly add method summary.

File: samples/tab-deeplink/csharp/DeepLinkBot/Services/DeeplinkHelper.cs [hunk 2]
```diff
@@ -25,6 +25,22 @@ public string GetDeepLinkToTabTask(string teamsUrl, string appID,string entityId
...
+
+            string jsoncontext = sb.ToString();
+            string taskContext = HttpUtility.UrlEncode(jsoncontext);
+            string encodedUrl = HttpUtility.UrlEncode(baseUrl + "/appInMeeting");
+            string deepLinkURL = teamsUrl + appID + "/" + entityId + "?webUrl=" + encodedUrl + "&context=";
+            string sidePanelDeepLink = deepLinkURL + taskContext;
+            return sidePanelDeepLink;
+        }
+
         public string GetDeepLinkToChannelTask(string teamsUrl, string appID,string baseUrl,string channelId, string entityId, string subEntityID)
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/csharp/DeepLinkBot/Services/DeeplinkHelper.cs:44]: Kindly add method summary.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments

File: samples/tab-deeplink/csharp/DeepLinkBot/Services/DeeplinkHelper.cs (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/csharp/DeepLinkBot/Services/DeeplinkHelper.cs:44]: Added

File: samples/tab-deeplink/csharp/DeepLinkBot/Services/DeeplinkHelper.cs (hunk 2 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/csharp/DeepLinkBot/Services/DeeplinkHelper.cs]: added

**Comment by ChetanSharma-msft** [comment]
@Prithvi-MSFT - Kindly run build pipeline before merging this PR.

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5066762364

**Pull request merged**